tests/functional/test_report_only_no_errors.py

```diff
@@ -0,0 +1,319 @@
+            test_report)
+
+    test_repo = os.environ.get("TEST_REPO")
+    if not test_repo:
+        raise Exception("TEST_REPO environment variable not defined")
+    fork_repo = os.environ.get("FORK_REPO")
+    if not fork_repo:
+        raise Exception("FORK_REPO environment variable not defined")
+    bot_name = fork_repo.split("/")[0]
+    cluster_token = os.environ.get("CLUSTER_TOKEN")
+    if not cluster_token:
+        raise Exception("CLUSTER_TOKEN environment variable not defined")
+    bot_token = os.environ.get("BOT_TOKEN")
+    if not bot_token:
+        bot_name = "github-actions[bot]"
+        bot_token = os.environ.get("GITHUB_TOKEN")
+        if not bot_token:
+            raise Exception("BOT_TOKEN environment variable not defined")
+
+    pr_base_branch = 'test-charts'
+    fork_branch = pr_base_branch + '-pr'
+    test_repo = str(base64.b64decode(test_repo), encoding="utf-8")
+    fork_repo = str(base64.b64decode(fork_repo), encoding="utf-8")
+    yield Secret(test_repo, fork_repo, cluster_token, bot_token, bot_name, pr_base_branch, fork_branch)
+
+    # Teardown step to cleanup branches
+    headers = {'Accept': 'application/vnd.github.v3+json',
+               'Authorization': f'Bearer {bot_token}'}
+    requests.delete(
+        f'https://api.github.com/repos/{test_repo}/git/refs/heads/{pr_base_branch}-gh-pages', headers=headers)
+    requests.delete(
+        f'https://api.github.com/repos/{fork_repo}/git/refs/heads/{fork_branch}', headers=headers)
+
+
+@scenario('features/report_without_chart.feature', "The partner hashicorp submits a error-free report for the vault chart")
+def test_partner_submits_report_without_any_errors():
+    """The partner hashicorp submits a error-free report for the vault chart."""
+
+
+@scenario('features/report_without_chart.feature', "The redhat associate submits a error-free report for the vault chart")
+def test_redhat_submits_report_without_any_errors():
+    """The redhat associate submits a error-free report for the vault chart."""
+
+
+@given("hashicorp is a valid partner")
+def hashicorp_is_a_valid_partner(secrets):
+    secrets.vendor_type = 'partners'
+    secrets.vendor = 'hashicorp'
+
+
+@given("a redhat associate has a valid identity")
+def hashicorp_is_a_valid_partner(secrets):
+    secrets.vendor_type = 'redhat'
+    secrets.vendor = 'redhat'
+
+
+@given("hashicorp has created a error-free report")
+@given("the redhat associate has created a report without any errors")
+def the_user_has_created_a_report_without_errors(secrets):
+    """The user has created a report without any errors."""
+
+    repo = git.Repo(os.getcwd())
+    if os.environ.get('WORKFLOW_DEVELOPMENT'):
+        logger.info("Wokflow development enabled")
+        repo.git.add(A=True)
+        repo.git.commit('-m', 'Checkpoint')
+
+    headers = {'Accept': 'application/vnd.github.v3+json',
+               'Authorization': f'Bearer {secrets.bot_token}'}
```

Comment:
I don't want to depend on a bot running to be able to run this test, can we override this with our identity?

Reply:
yes we can, we just need to set the `bot_token` with our personal access token
for testing I was using my bot account (which is just another github user): https://github.com/abai-test-bot

Reply:
OK, good